src/utils/contacts.js

```diff
@@ -241,6 +238,10 @@ async function remove({ userId, contact }) {
   pipe.del(key);
   pipe.srem(redisKey(userId, USERS_CONTACTS), contact.value);
 
+  if (this.config.contacts.updateUsername) {
+    pipe.hdel(USERS_USERNAME_TO_ID, contact.value);
```

Comment:
this doesn't seem right. you always need a username -> id mapping and this deletes it?